charts/cloudzero-agent/README.md

```diff
@@ -17,255 +17,36 @@ For the latest release, see [Releases](https://github.com/Cloudzero/cloudzero-ch
 
 ## Installation
 
-### Adding Helm Repository Information
+### Easy Install (Most teams will use this!)
 
 To use the chart or a beta version, you must add the repository to Helm. Refer to the [`helm repo`](https://helm.sh/docs/helm/helm_repo/) documentation for command details.
 
-#### 1. Add the Helm Chart Repository
+#### 1. Install the Helm Chart
 
 ```console
 helm repo add cloudzero https://cloudzero.github.io/cloudzero-charts
+helm install cloudzero cloudzero/cloudzero-agent \
+    --set apiKey=<YOUR_CLOUDZERO_API_KEY>\
+    --set clusterName=<YOUR_CLUSTER_NAME>
 ```
-> Note: If you intend to use a beta version, refer to the [beta installation document](./BETA-INSTALLATION.md) for the appropriate channel.
 
-#### 2. Add Repository Dependencies
+### Advanced Install (Usually when you have specific security requirements.)
 
-**Cert Manager**
```

Comment:
Do we need to ship subcharts? I would suggest going ahead and highly tuning the prometheus and KSM image we want to ship so we're not trying to pass details or manage subcharts. IE, a customer doesn't necessarily care that we are shipping a subchart provided by prometheus, they're still going to hold us accountable for the image that we're shipping.